autoload/pydocstring.vim

```diff
@@ -73,6 +73,7 @@ function! s:builddocstring(strs, indent)
     let docstrings = []
     let lines = s:readtmpl('multi')
     for line in lines
+      let a:indent = repeat(' ', %shiftwidth, '') . a:indent 
```

Comment:
Here I add a number of spaces equal to shiftwidth to indent the docstring another level, instead of inserting it at the indentation level of the function definition.